docker/1.11.0/final/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,64 @@
+FROM tensorflow/serving:1.11.0 as tensorflow_serving_image
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+ARG framework_installable
+ARG framework_support_installable=sagemaker_tensorflow_container-1.0.0.tar.gz
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    python \
+    curl \
+    nginx \
+ && rm -rf /var/lib/apt/lists/*
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
+
+# Removing tests to free some space
+RUN set -ex; \
+	curl -k -O https://bootstrap.pypa.io/get-pip.py && \
+	python get-pip.py \
+		--disable-pip-version-check \
+		--no-cache-dir \
+		"pip==18.1" \
+	; \
+	pip --version; \
+	find /usr/local -depth \
+		\( \
+			\( -type d -a \( -name test -o -name tests \) \) \
+			-o \
+			\( -type f -a \( -name '*.pyc' -o -name '*.pyo' \) \) \
+		\) -exec rm -rf '{}' +; \
+	rm -f get-pip.py
+
+WORKDIR /
+
+# Install TF Serving pkg
+COPY --from=tensorflow_serving_image /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+COPY $framework_installable .
+COPY $framework_support_installable .
+
+RUN pip install --no-cache-dir \
+    numpy \
+    scipy \
+    sklearn \
+    pandas \
+    Pillow \
+    h5py \
+    tensorflow-serving-api==1.11.0 \
+    \
+    $framework_installable \
```

Comment:
before we had issue with tensorflow binaries already being installed as a serving dependency (as far as i remember).
Should we add --upgrade to make sure our mkl optimized binaries are installed or move framework installation earlier?